File: python programming/forloop_2.py
n=int(input("enter no of term:-"))
sum=1
print("1",end='+')
for i in range(1,n):
 print(f'{i+1}/{i}',end='')
 if i<n-1:
  print('+',end='')
 sum=sum+(i+1)/i
print("sum of seriese=",sum)

File: JavaNIDE/Javaprgm/app/src/main/java/sample/Forloop2.java
package sample;
import java.util.*;

class Forloop2 
{
  
  public static void main (String[] ar)
  {
    int n,i,sum=0,m;
    System.out.println("enter no of term");
    Scanner z=new Scanner(System.in);
    n=z.nextInt();
    for(i=1;i<=n;i++)
    {
      m=i*i;
      System.out.print(i+"^2");
      if(i<n)
      {
        System.out.print("+");
      }
     sum=sum+m;
      
    }
    System.out.print("\nsum of seriese="+sum);
  }
}

